include/picolibrary/testing/unit/indicator.h

```diff
@@ -23,10 +23,26 @@
 #ifndef PICOLIBRARY_TESTING_UNIT_INDICATOR_H
 #define PICOLIBRARY_TESTING_UNIT_INDICATOR_H
 
+#include "picolibrary/indicator.h"
+#include "picolibrary/testing/unit/random.h"
+
 /**
  * \brief Indicator unit testing facilities.
  */
-namespace picolibrary::Testing::Unit::Indicator {
-} // namespace picolibrary::Testing::Unit::Indicator
```

Comment:
The indicator namespace should not have been deleted since things may be (will be in this case) added to it later. See https://github.com/apcountryman/picolibrary/blob/master/include/picolibrary/testing/unit/gpio.h for an example of how to structure a unit testing header file that uses multiple namespaces (the more general namespace and its content appears before the more specific namespace).